nncf/experimental/openvino_native/hardware/fused_patterns.py

```diff
@@ -399,6 +399,24 @@ def create_hswish_without_denominator():
     return pattern
 
 
+@OPENVINO_HW_FUSED_PATTERNS.register(PatternNames.SWISH_WITH_HARD_SIGMOID)
+def create_swish_with_hardsigmoid():
```

Comment:
Why did you add `swish_with_hardsigmoid` pattern only for OpenVINO backend?

Reply:
please merge it as it is and open a new ticket for other BEs

Reply:
ticket: 109256